bin/aws_access

```diff
@@ -3,6 +3,16 @@
 require_relative '../deployment'
 require 'aws-sdk-sts'
 
+module Aws
+  class CredentialProviderChain
+    def current_provider
+      providers.find do |method_name, options|
+        send(method_name, options.merge(config: @config))&.set?
```

Comment:
Everything makes sense to me except this - why does this behavior have to be overridden now?

Reply:
This behavior-override was only moved, from its previous location in `cdo_google_credentials.rb` more narrowly into `bin/aws_access` (which is the only script that uses it).

Previously the `Aws::CredentialProviderChain` class was overridden for a second purpose, which was moved into the `aws-google` gem to [`credential_provider.rb`](https://github.com/code-dot-org/aws-google/blob/master/lib/aws/google/credential_provider.rb). So now that there's only one behavior-override left in this codebase, it made more sense to move it directly into the script.